science/libsbml/Portfile

```diff
@@ -1,12 +1,13 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
+PortGroup           cmake 1.1
 
 name                libsbml
-version             5.13.0
+version             5.16.0
 categories          science
 platforms           darwin
-maintainers         nomaintainer
+maintainers         @funasoul openmaintainer
```

Comment:
Thanks for taking over as the maintainer. You should include both your github account name and your obfuscated email address so that you can get notifications about things like failures of automatic builds. Instructions are in the macports documentation.

Reply:
Thanks, I've added my email address and squashed my commits into one commit.